package.json

```diff
@@ -53,7 +53,7 @@
     "joi": "^17.1.1",
     "js-yaml": "^3.13.0",
     "keymbinatorial": "^1.1.5",
-    "screwdriver-data-schema": "^20.0.0",
+    "screwdriver-data-schema": "^20.2.2",
```

Comment:
Probably, v20.2.2 will be created once https://github.com/screwdriver-cd/data-schema/pull/414 is merged.